README.md

```diff
@@ -300,6 +302,31 @@ plugins:
       visitorData: "paste your visitor_data here"
 ```
 
+## Using a remote cipher server
+
+It becomes harder and harder to keep up with YouTube's cipher changes, as they become more frequent and complex.
+To help with this, you can use a remote cipher server to handle signature deciphering for you.
+You can use [yt-cipher](https://github.com/kikkia/yt-cipher), which is a simple Deno server that exposes a REST API for deciphering signatures.
+Check out the repository for more information on how to set it up.
+
+If you want to implement your own, you can follow the [yt-cipher API specification](https://github.com/kikkia/yt-cipher#api-specification).
```

Comment:
I feel like the specification should be determined by youtube-source and not an implementation of, but this is just a nit/thought.